src/app/shared/components/tooltip/tooltip.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="c-sofie-tooltip-wrapper hidden" #tooltipElementWrapper>
+  <div class="c-sofie-tooltip-content" #tooltipElementContent>
```

Comment:
Do you really need two divs?

Reply:
@LindvedKrvang Yes I had a problem with how the html behaves when we use it an absolute position and make it relative size from a point to the end of the screen. In such situations when we answered the window it extended from our point only to the edge of the screen. To solve this I change element to be from point 0 to the end of the screen with absolute position, and the second is with a relative position located somewhere in the first.